### ISSUE TITLE
Copy the spec tests for aggregates

### ISSUE THREAD

**Issue by wschella** [comunica/sparqlee#41]
As we did for the spec tests for functions, we should copy the spec tests for aggregates from the test-repo.

**Comment by jitsedesmet** [comment]
@wschella or @rubensworks, do you have a reference on where to find these test?

**Comment by rubensworks** [comment]
Probably these ones: https://github.com/w3c/rdf-tests/tree/main/sparql11/data-sparql11/aggregates

**Comment by jitsedesmet** [comment]
Since these are also a lot of files, I will split the spec directory.

**Comment by jitsedesmet** [comment]
I've been trying for quite some time now. I have no idea on how to test this within sparqlee. 
Take for example the [agg2 test](https://github.com/w3c/rdf-tests/blob/main/sparql11/data-sparql11/aggregates/agg02.rq). If I'm not mistaken we have a whole actor in comunica to handle these kind of combinations? How would we go about testing this here?

**Comment by rubensworks** [comment]
I don't think the intention was to copy the tests as-is. But instead, to select the elements of these tests that are applicable to sparqlee, i.e., those that interact directly with the aggregate evaluator.

**Comment by jitsedesmet** [comment]
So we would just exclude the `agg2` test? because I don't think we can test this.

**Comment by rubensworks** [comment]
You'll know better than me, but can't we test the `COUNT` part?
If I remember correctly, grouping is done purely on the Comunica side, so that can't be tested in Sparqlee indeed.

**Comment by jitsedesmet** [comment]
Yes, I guess we could. I'll write the tests. Doesn't really seem all that valuable to me but I guess it's nice to have.
